Source/WebCore/page/Quirks.h

```diff
@@ -249,6 +250,7 @@ class Quirks {
     mutable std::optional<bool> m_needsFullscreenObjectFitQuirk;
     mutable std::optional<bool> m_shouldAvoidPastingImagesAsWebContent;
     mutable std::optional<bool> m_mayNeedToIgnoreContentObservation;
+    mutable std::optional<bool> m_needsGoogleMapsScrollQuirk;
```

Comment:
"scroll" or "scrolling"? not sure why the inconsistency

Reply:
Will fix the inconsistency!

Reply:
I think we should use "scrolling"? Like `needsWeChatScrollingQuirk`.